src/scene/export.ts

```diff
@@ -94,11 +94,11 @@ export function exportToSvg(
     <style>
       @font-face {
         font-family: "Virgil";
-        src: url("https://excalidraw.com/FG_Virgil.woff2");
+        src: url("https://excalidraw.com/static/fonts/FG_Virgil.woff2");
```

Comment:
Wait, you can’t do that. You just broke all the svgs that were exported as we hardcode the url

Reply:
Having a new URL for new exports is fine as long as the OLD url still works.

Reply:
In order to make both the old and the new one work, we need duplicate the fonts which doesn't seem to be fair :(

cc: @kbariotis

Reply:
Indeed, we will redirect the old fonts to keep the old paths working. The preloading is the thing that still remains and we may need another solution. This whole thing happened because we wanted `webpack` to parse all assets in order to generate a valid `manifest.json` for offline caching.

Simplest thing that comes to my mind is to separately preload the fonts but let the service worker download a cache a different version.